init.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+pr_blue() {
+    echo -e "\e[1;34m$*\e[0m"
+}
+
+TOKEN=""
```

Comment:
is there any way we can use `.env` here and fetch all these from there.

Reply:
Actually we can do this but we have something similar already implemented, like currently this `init.sh` script take all variable and store in a file called `.token.sh` and whenever bot in started with `./bot.sh` it takes configs from this file, so in short we just need to rename the `.token.sh` to `config.env`